lightningd/plugin.c

```diff
@@ -974,11 +986,11 @@ void plugins_init(struct plugins *plugins, const char *dev_plugin_debug)
 	plugins->pending_manifests = 0;
 	uintmap_init(&plugins->pending_requests);
 
-	plugins_add_default_dir(plugins, path_join(tmpctx, plugins->ld->config_dir, "plugins"));
-
-	setenv("LIGHTNINGD_PLUGIN", "1", 1);
```

Comment:
Why is env removed?  In theory we do this so that a plugin could be run outside of `lightningd` (it could do something as simple as print "this is a lightningd plugin, not intended to be run directly as a command").

Reply:
This was not intended